### DISCUSSION TITLE
请问抖音的数字user_id能获取到用户信息吗？非sec_userid

### DISCUSSION THREAD

**Discussion by biexiaozhi** [Johnserf-Seed/TikTokDownload#527] [Q&A] [answered]
纯技术交流，不会拿来做非法用途，只是想知道一个弹幕是谁发的QWQ

**Comment by Johnserf-Seed** [comment, marked as answer]
```web```的接口可以通过```/aweme/v1/web/discover/search/```搜索```抖音号(unique_id)```和 ```/aweme/v1/web/user/profile/other/```搜索```sec_user_id```来查询。```uid```和```short_uid```暂未发现接口

**Reply by biexiaozhi** [reply]
好吧，谢谢！
